mm2src/mm2_libp2p/src/atomicdex_behaviour.rs

```diff
@@ -470,31 +543,34 @@ pub fn start_gossipsub(
             MessageId(s.finish().to_string())
         };
 
+        let (mesh_n_low, mesh_n, mesh_n_high) = if i_am_relay { (4, 6, 12) } else { (2, 3, 4) };
+
         // set custom gossipsub
         let gossipsub_config = GossipsubConfigBuilder::new()
             .message_id_fn(message_id_fn)
             .i_am_relay(i_am_relay)
+            .mesh_n_low(mesh_n_low)
+            .mesh_n(mesh_n)
+            .mesh_n(mesh_n_high)
```

Comment:
It looks like [mesh_n_high](https://github.com/KomodoPlatform/atomicDEX-API/blob/libp2p-simple-peer-discovery/mm2src/gossipsub/src/config.rs#L177) should be called here :)